src/release-pr.ts

```diff
@@ -128,29 +128,31 @@ export class ReleasePR {
     this.changelogSections = options.changelogSections;
   }
 
-  async run() {
+  async run(): Promise<number | undefined> {
     if (this.snapshot && !this.supportsSnapshots()) {
       checkpoint(
         'snapshot releases not supported for this releaser',
         CheckpointType.Failure
       );
       return;
     }
-    const pr: GitHubReleasePR | undefined = await this.gh.findMergedReleasePR(
-      this.labels
-    );
-    if (pr) {
+    const mergedPR:
+      | GitHubReleasePR
+      | undefined = await this.gh.findMergedReleasePR(this.labels);
+    if (mergedPR) {
       // a PR already exists in the autorelease: pending state.
       checkpoint(
-        `pull #${pr.number} ${pr.sha} has not yet been released`,
+        `pull #${mergedPR.number} ${mergedPR.sha} has not yet been released`,
         CheckpointType.Failure
       );
+      return undefined;
     } else {
-      return this._run();
+      const openedPR = this._run();
+      return openedPR ? openedPR : undefined;
```

Comment:
Huh - is this trying to normalize null and undefined?

Reply:
we return `0` from our github helper, I figured for the benefit of the GitHub action `undefined` would be a better signal that we didn't do anything.

I suppose we could switch to `undefined` everywhere.

Reply:
switched to returning `undefined` everywhere.